.env.sample

```diff
@@ -5,7 +5,7 @@ DS_API_TOKEN=SUPERSECRET
 DATABASE_URL=postgres://docker:docker@localhost:5400/api-dev
 
 # do not set the NODE_ENV. It is 'development' by default. 
-# Production will be set by env vars.
```

Comment:
Please re-add this line

Reply:
I didn't do any of these changes

Reply:
Gotcha. Well, since this doesn't match what's in main right now, I'd like you to run `git checkout origin/main .env.sample` to restore this file to its main version.